Changing the language

**Describe the bug**
After I changed the language, the language icons don't appear anymore. Got this error:

> vue.esm.js:1741 
Error: Cannot find module './en.svg'
    at i (.*\.svg$:18)
    at a (.*\.svg$:12)
    at s.getLanguageFlag (Desktop.vue:41)
    at Desktop.vue?7f9f:1
    at s.Et [as _l] (vue.esm.js:3705)
    at s.<anonymous> (Desktop.vue?7f9f:1)
    at s.e._render (vue.esm.js:4544)
    at s.<anonymous> (vue.esm.js:2788)
    at Yt.get (vue.esm.js:3142)
    at new Yt (vue.esm.js:3131)

**To Reproduce**
Steps to reproduce the behavior:
1. Go to https://explorer.ark.io/
2. Click on the globe icon top right and change the language to GB
3. Reload the page
4. Click on the globe icon top right

**Screenshots**
<img width="1231" alt="screenshot 2018-12-13 at 10 30 24" src="https://user-images.githubusercontent.com/3462869/49925826-aaf39200-fec2-11e8-9f36-6459d4cbbef0.png">
<img width="1223" alt="screenshot 2018-12-13 at 10 30 30" src="https://user-images.githubusercontent.com/3462869/49925819-a7f8a180-fec2-11e8-8462-8e50018c8c2c.png">
<img width="1280" alt="screenshot 2018-12-13 at 10 38 14" src="https://user-images.githubusercontent.com/3462869/49926001-3836e680-fec3-11e8-8fb4-7af90dbc6e03.png">

**Server (please complete the following information):**
 - OS: macOS Mojave 10.14.1
 - Browser: Google Chrome 70.0.3538.110 (Official Build) (64-bit)

--- a/src/components/header/languages/Desktop.vue
+++ b/src/components/header/languages/Desktop.vue
@@ -20,7 +20,7 @@ export default {
 
     languages() {
       return Object.keys(this.$i18n.messages).filter(
-        l => l != this.language
+        lang => lang !== this.language && lang !== 'en'
       )
     }
   },

--- a/src/components/header/languages/Mobile.vue
+++ b/src/components/header/languages/Mobile.vue
@@ -21,7 +21,7 @@ export default {
 
     languages() {
       return Object.keys(this.$i18n.messages).filter(
-        l => l != this.language
+        lang => lang !== this.language && lang !== 'en'
       )
     }
   },

